tests.py

```diff
@@ -849,6 +849,13 @@ def test_implicit_wrapInstance():
         finally:
             app.exit()
 
+    def test_isValid():
+        from Qt import QtCompat, QtCore
+        obj = QtCore.QObject()
+        assert QtCompat.isValid(obj)
+        QtCompat.delete(obj)
+        assert not QtCompat.isValid(obj)
```

Comment:
This test is only running on Python 3.4 and before, but technically it works for all the `docker run` commands I see in your README. I just put this next to `wrapInstance()`, no idea if that was the right thing.

Reply:
I think that's right; in the comment above that block, it says shiboken1 isn't available prior to that, and this test would depend on it.